src/liboslexec/llvm_util.cpp

```diff
@@ -3637,12 +3646,18 @@ LLVM_Util::call_function(llvm::Value* func, cspan<llvm::Value*> args)
 llvm::Value*
 LLVM_Util::call_function(const char* name, cspan<llvm::Value*> args)
 {
-    llvm::Function* func = module()->getFunction(name);
+    ustring function_prefix;
```

Comment:
Seeing this pattern multiple times, suggest add a helper build_prefixed_functionname(...) to better encapsulate it

Reply:
Perhaps "prefix" is a bit too specific, basically the target wants to translated function names, perhaps a more generic translation layer than just "prefix".